org.hl7.fhir.convertors/src/test/java/org/hl7/fhir/convertors/conv10_40/MedicationRequest10_40Test.java

```diff
@@ -0,0 +1,109 @@
+            "      'end': '2016-11-23T00:00:00Z'}}},"+
+            "   'doseQuantity': {'value': 10.0,"+
+            "    'unit': 'mg',"+
+            "    'code': 'mg',"+
+            "    'system': 'http://unitsofmeasure.org'}}],"+
+            " 'dispenseRequest': {'validityPeriod': {'start': '2016-11-15T00:00:00Z',"+
+            "   'end': '2016-11-23T00:00:00Z'}}}";
+        String r4_expected_string =
+            "{\"resourceType\": \"MedicationRequest\","+
+            " \"id\": \"T5YI1tCzs--JEvCICFbx8zgB\","+
+            " \"identifier\": [{\"use\": \"usual\","+
+            "   \"system\": \"urn:oid:1.2.840.114350.1.13.0.1.7.2.798268\","+
+            "   \"value\": \"988736\"},"+
+            "  {\"use\": \"usual\","+
+            "   \"system\": \"urn:oid:1.2.840.114350.1.13.0.1.7.3.798268.801\","+
+            "   \"value\": \"988736:2150291843\"}],"+
+            " \"status\": \"active\","+
+            " \"intent\": \"order\","+
+            " \"medicationReference\": {\"reference\": \"https://open-ic.epic.com/Argonaut/api/FHIR/DSTU2/Medication/T0eKLT7EB2ApMM8HCEURdMAB\","+
+            "  \"display\": \"amitriptyline 10 MG tablet\"},"+
+            " \"subject\": {\"reference\": \"https://open-ic.epic.com/Argonaut/api/FHIR/DSTU2/Patient/Tbt3KuCY0B5PSrJvCu2j-PlK.aiHsu2xUjUM8bWpetXoB\","+
+            "  \"display\": \"Jason Argonaut\"},"+
+            " \"authoredOn\": \"2016-11-13T00:00:00\","+
+            " \"requester\": {\"reference\": \"https://open-ic.epic.com/Argonaut/api/FHIR/DSTU2/Practitioner/T-kmjPGEVPAmnBfmx56HsKgB\","+
+            "  \"display\": \"Historical Provider, MD\"},"+
+            " \"dosageInstruction\": [{\"text\": \"Take 10 mg by mouth nightly.\","+
+            "   \"timing\": {\"repeat\": {\"boundsPeriod\": {\"start\": \"2016-11-15T00:00:00Z\","+
+            "      \"end\": \"2016-11-23T00:00:00Z\"},"+
+            "     \"count\": 0,"+
+            "     \"frequency\": 1,"+
+            "     \"frequencyMax\": 0,"+
+            "     \"period\": 1.0,"+
+            "     \"periodUnit\": \"d\"}},"+
+            "   \"asNeededBoolean\": \"False\","+
+            "   \"route\": {\"coding\": [{\"system\": \"urn:oid:1.2.840.114350.1.13.0.1.7.4.698288.330\","+
+            "      \"code\": \"15\","+
+            "      \"display\": \"Oral\"}],"+
+            "    \"text\": \"Oral\"},"+
+            "   \"method\": {\"coding\": [{\"system\": \"urn:oid:1.2.840.114350.1.13.0.1.7.4.798268.8600\","+
+            "      \"code\": \"11\","+
+            "      \"display\": \"Take\"}],"+
+            "    \"text\": \"Take\"},"+
+            "   \"doseAndRate\": [{\"doseQuantity\": {\"value\": 10.0,"+
+            "      \"unit\": \"mg\","+
+            "      \"system\": \"http://unitsofmeasure.org\","+
+            "      \"code\": \"mg\"}}]}],"+
+            " \"dispenseRequest\": {\"validityPeriod\": {\"start\": \"2016-11-15T00:00:00Z\","+
+            "   \"end\": \"2016-11-23T00:00:00Z\"}}}";
```

Comment:
please put two strings into separate *.json resource files, and then load them with the class loader

Reply:
fixed it - looks better now
thanks